src/tests/conftest.py

```
@@ -0,0 +1,72 @@
+See https://docs.pytest.org/en/stable/reference/fixtures.html
+"""
+
+import pytest
+
+
+@pytest.fixture
+def logged_in_client(client, django_user_model):
+    """Return a client with an authenticated user."""
+    user = django_user_model.objects.create_user(
+        username="testuser",
+        password="testpass123",
```

Ruff (0.14.10)

Possible hardcoded password assigned to argument: "password"

(S106)